src/utils/components/CloudinitModal/utils/consts.ts

```diff
@@ -3,3 +3,6 @@ export const CLOUD_CONFIG_HEADER = '#cloud-config';
 export const ACTIVATION_KEY = 'activation-key';
 export const ORG_KEY = 'org';
 export const RH_SUBSCRIPTION_KEY = 'rh_subscription';
+
+export const AUTO_UPDATE_OS_CMD = 'systemctl enable --now dnf-automatic-install.timer';
+export const DNF_AUTOMATIC_PACKAGE = 'dnf-automatic';
```

Comment:
Maybe we could rename this `consts.ts` file no nicer _constants.ts_, as we usually do everywhere else?